tools/vargo/src/main.rs

```diff
@@ -296,6 +296,16 @@ fn run() -> Result<(), String> {
             });
     }
 
+    let local_store = args_bucket
+        .iter()
+        .position(|x| x.as_str() == "--local-store")
+        .map(|p| args_bucket.remove(p))
+        .is_some();
+
+    if local_store {
+        std::env::set_var("VERUS_LOCAL_STORE", "1");
+    }
+
```

Comment:
I recommend removing this entirely. We shouldn't be touching vargo for this feature.